quickstarts/key-vault/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "service-bus-with-typescript",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "lint": "eslint . --ext .ts --fix",
+    "prettier:fix": "prettier --write \"src/**/*.ts\""
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
```

Comment:
It's probably not a bad idea to populate these fields, even if the package isn't being published to npm. Particularly the license. See the [Secrets SDK package package.json](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-secrets/package.json#L4-L7) for an example.